examples/games_fps.html

```diff
@@ -210,7 +210,11 @@
 
 					}
 
-					playerCollider.translate( result.normal.multiplyScalar( result.depth ) );
+					if ( result.depth >= 1e-10 ) {
```

Comment:
How did you come up with this specific value `1e-10`?

Reply:
`triangleCapsuleIntersect` uses `lineToLineClosestPoints`:

https://github.com/mrdoob/three.js/blob/88f8b7c03cf40b0cd2e8d126efd2dc1a992feceb/examples/jsm/math/Octree.js#L262

where `lineToLineClosestPoints` uses `EPS`:

https://github.com/mrdoob/three.js/blob/88f8b7c03cf40b0cd2e8d126efd2dc1a992feceb/examples/jsm/math/Octree.js#L43

where `EPS` was defined as `1e-10`:

https://github.com/mrdoob/three.js/blob/88f8b7c03cf40b0cd2e8d126efd2dc1a992feceb/examples/jsm/math/Octree.js#L26

Reply:
Okay. It clearly fixes the issue and defining epsilon values is always somewhat controversial^^. Let's give this a try.

Reply:
ok.